### ISSUE TITLE
Problems on home assistant first start-up

### ISSUE THREAD

**Issue by YuriTAngelico** [home-assistant/home-assistant.github.io#24915]
### Feedback

Hello everyone!

So I followed all the instructions from this web page: [Install Home Assistant Core](https://www.home-assistant.io/installation/raspberrypi), but when I started the home assistant for the first time using the command "hass" with my venv activated i had this error:

````
(homeassistant) pi@raspberrypi:/srv/homeassistant $ hass
2022-11-12 17:50:35.291 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: recorder
2022-11-12 17:54:33.346 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: bluetooth
2022-11-12 17:55:52.085 ERROR (MainThread) [homeassistant.setup] Setup failed for cloud: Unable to import component: 

IMPORTANT: PLEASE READ THIS FOR ADVICE ON HOW TO SOLVE THIS ISSUE!

Importing the numpy C-extensions failed. This error can happen for
many reasons, often due to issues with your setup or how NumPy was
installed.

We have compiled some common reasons and troubleshooting tips at:

    https://numpy.org/devdocs/user/troubleshooting-importerror.html

Please note and check the following:

  * The Python version is: Python3.9 from "/srv/homeassistant/bin/python3"
  * The NumPy version is: "1.23.2"

and make sure that they are the versions you expect.
Please carefully study the documentation linked above for further help.

Original error was: libcblas.so.3: cannot open shared object file: No such file or directory

2022-11-12 17:55:53.329 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: input_button, input_number, input_datetime, input_select, zone, input_boolean, schedule, sun, input_text, automation, script, application_credentials
2022-11-12 17:56:50.701 ERROR (MainThread) [homeassistant.setup] Setup failed for mobile_app: Unable to import component: 

IMPORTANT: PLEASE READ THIS FOR ADVICE ON HOW TO SOLVE THIS ISSUE!

Importing the numpy C-extensions failed. This error can happen for
many reasons, often due to issues with your setup or how NumPy was
installed.

We have compiled some common reasons and troubleshooting tips at:

    https://numpy.org/devdocs/user/troubleshooting-importerror.html

Please note and check the following:

  * The Python version is: Python3.9 from "/srv/homeassistant/bin/python3"
  * The NumPy version is: "1.23.2"

and make sure that they are the versions you expect.
Please carefully study the documentation linked above for further help.

Original error was: libcblas.so.3: cannot open shared object file: No such file or directory

2022-11-12 17:56:50.710 ERROR (MainThread) [homeassistant.setup] Unable to set up dependencies of default_config. Setup failed for dependencies: cloud, mobile_app
2022-11-12 17:56:50.713 ERROR (MainThread) [homeassistant.setup] Setup failed for default_config: (DependencyError(...), 'Could not setup dependencies: cloud, mobile_app')
````

But after some research about the original error described I found a simple solution:

````
sudo apt-get install libatlas-base-dev
````

Found here: [[ImportError: libcblas.so.3: cannot open shared object file: No such file or directory](https://stackoverflow.com/questions/53347759/importerror-libcblas-so-3-cannot-open-shared-object-file-no-such-file-or-dire)](https://stackoverflow.com/questions/53347759/importerror-libcblas-so-3-cannot-open-shared-object-file-no-such-file-or-dire)

**So that is not really an issue is more a something that I found and corrected. I still did not configured the home assistant with nothing. But if I ran into more issues I let you known.**

### URL

https://www.home-assistant.io/installation/raspberrypi

### Version

homeassistant 2022.11.2

### Additional information

My setup:

Raspberry PI 3 Model B+ (1GB of RAM)

**Comment by frenck** [comment]
The error messages your Python raises are pretty clear:

```
IMPORTANT: PLEASE READ THIS FOR ADVICE ON HOW TO SOLVE THIS ISSUE!

Importing the numpy C-extensions failed. This error can happen for
many reasons, often due to issues with your setup or how NumPy was
installed.

We have compiled some common reasons and troubleshooting tips at:

    https://numpy.org/devdocs/user/troubleshooting-importerror.html

Please note and check the following:

  * The Python version is: Python3.9 from "/srv/homeassistant/bin/python3"
  * The NumPy version is: "1.23.2"

and make sure that they are the versions you expect.
Please carefully study the documentation linked above for further help.

Original error was: libcblas.so.3: cannot open shared object file: No such file or directory
```

I'm not sure what the suggestion is you are trying to raise, or if you are seeking for support or maybe even report an issue? Could you clarify?

**Comment by YuriTAngelico** [comment]
Hello @frenck .

In fact, I just wanted to bring the solution I found to the problem, because that suggestion didn't work for me and for many people who tried to install the home assistant core.
So, in fact, I didn't want to raise a problem, but to bring a solution that I found after a lot of searching.

Because this solution, I tried but I can't in any way finish the installation.

And as I said at the end of my comment the solution I found was:

`
sudo apt-get install libatlas-base-dev
`

Found here: [[ImportError: libcblas.so.3: cannot open shared object file: No such file or directory](https://stackoverflow.com/questions/53347759/importerror-libcblas-so-3-cannot-open-shared-object-file-no-such-file-or-dire)](https://stackoverflow.com/questions/53347759/importerror-libcblas-so-3-cannot-open-shared-object-file-no-such-file-or-dire)

So that is not really an issue is more a something that I found and corrected. I still did not configured the home assistant with nothing. But if I ran into more issues I let you known.
